core/src/sigverify_stage.rs

```diff
@@ -177,11 +199,13 @@ impl SigVerifyStage {
         let (mut batches, len, recv_time) = streamer::recv_batch(recvr)?;
 
         let mut verify_batch_time = Measure::start("sigverify_batch_time");
+        let mut discard_time = Measure::start("sigverify_discard_time");
         let batches_len = batches.len();
         debug!("@{:?} verifier: verifying: {}", timing::timestamp(), len,);
         if len > MAX_SIGVERIFY_BATCH {
             Self::discard_excess_packets(&mut batches, MAX_SIGVERIFY_BATCH);
         }
+        discard_time.stop();
```

Comment:
Move the verify time to just around the verify_batches?

Reply:
for the release.  i think if we hack it i would rather have the cluster report the same value for that variable.